app/views/development/development.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title do %>Routes<% end %>
```

Comment:
Where did this come from? It's a nice idea, but's it's another thing that needs to be kept up to date 😞

Reply:
Argh, that was me seeing if it could be done automagically. Will delete!